### ISSUE TITLE
Replace of values at the end of a string is greedy with carriage returns

### ISSUE THREAD

**Issue by salimepoint** [jedld/multi_string_replace#3]
Hi guys,
Hopefully a fairly easy fix.. 

If a string ends with a value to replace and then a carriage return, `MultiStringReplace` is stripping the carriage return.

If a string includes a value to replace somewhere in the string, and that string ends with a carriage return, `MultiStringReplace` leaves the carriage return at the end of the string.

```ruby
irb(main):002:0> MultiStringReplace.replace("string ends with a replace\n", { 'replace': '123' })
=> "string ends with a 123"
irb(main):003:0> MultiStringReplace.replace("string has replace in it\n", { 'replace': '123' })
=> "string has 123 in it\n"
irb(main):004:0> 
```

Assuming this isn't expected functionality, could I ask that the carriage return remains in the string in both situations?

**Comment by jedld** [comment]
Thanks this should be fixed in 1.0.5

commit 79c92005b6d5124b8853f0024822b55c59183fa8

**Comment by salimepoint** [comment]
Thanks for the quick response Joseph! Amazing turnaround..
